README.md

```diff
@@ -1,2 +1,83 @@
 # BOE_tabulator
 Reads PDFs from Baltimore's archive of minutes from the Board of Estimates and places the data in a searchable table.
+
+## Table of Contents
+- [Overview](#overview)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+  - [Fetching the Data](#fetching-the-data)
+- [Usage](#usage)
+- [Contributing](#contributing)
+
+## Overview
+This is where the project overview will go  
+
+## Getting Started
+Follow the steps below to recreate the development environment necessary to start contributing to this project
+
+### Prerequisites
+- Python version 3.6 or later
+
+In order to check which version of python you have installed, run the following command in your command line (for Mac/Linux)
+
+>**NOTE:** in all of the code blocks below, lines preceded with `$` indicate commands you should enter in your command line (excluding the `$` itself), while lines preceded with `>` indicate the expected output from the previous command.
+
+```
+$ python --version
+```
+The output should look something like this:
+```
+> Python 3.7.7
+```
+If you don't have Python version 3.6 or later installed on your computer. Consider using [pyenv](https://github.com/pyenv/pyenv) to install and manage multiple versions of Python concurrently.
+
+### Installation
+1. Fork the repo -- for more information about forking, reference [this guide](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks)
```

Comment:
My only question here is whether we might add a section for users of Github Desktop, which is emerging as the standard in DGS. Plus, most city users will be on a PC, where Linux commands may not be readily available. 

For users of Github Desktop/Windows, there will be no command-line stuff. The flow will be more of a point-and-click process.

Reply:
Good point! Yeah I'm more of a command line user, but it would probably be worth splitting the steps into command line and GUI steps, or at least specifying both options in the git-specific steps of the installation